api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -992,19 +992,20 @@ def flag_unsafe_move(
         Raise an error if attempting to perform a move that's deemed unsafe due to
         the presence of the Heater-Shaker.
         """
+        destination_slot = to_loc.labware.first_parent()
+        if destination_slot is None:
+            MODULE_LOG.warning(
+                "Pipette movement destination has no slot associated with it. Cannot"
+                " determine movement safety to avoid colliding with the heater-shaker."
```

Comment:
Is this message user-facing? If so can we clean up the wording a little? (Either way I'd like to see `Heater-Shaker` capitalized 🙂)

```suggestion
                "Pipette movement destination has no slot associated with it. Cannot"
                " determine whether movement will safely avoid colliding with the Heater-Shaker."
```

Reply:
Not user-facing but happy to make that change